ext/json/json_parser.y

```diff
@@ -280,13 +280,16 @@ static int php_json_parser_object_update_validate(php_json_parser *parser, zval
 static int php_json_yylex(union YYSTYPE *value, php_json_parser *parser)
 {
 	int token = php_json_scan(&parser->scanner);
-	value->value = parser->scanner.value;
```

Comment:
I believe the problem radicates here, as value->value has to be set. I will check it locally

Reply:
I assumed it was zeroed on initialization since ASAN was happy but maybe not.

Reply:
let me check, i am trying something locally, I will be back in few minutes

Reply:
It was uninitialized, but only failed locally without ASAN. Weird.

Reply:
lets see what happens in your current PR checks pipes, I could not work it locally, parental life